.cirrus.yml

```diff
@@ -0,0 +1,63 @@
+# By default, Cirrus mounts an empty volume to `/tmp`
+# which triggers all sorts of warnings like "system temporary path is world-writable: /tmp".
+# Lets workaround it by specifying a custom volume mount point.
+env:
+  CIRRUS_VOLUME: /cirrus-ci-volume
+
+task:
+  # The entire testing design is inspired from .github/workflows/compilers.yml .
+  name: Arm64 Graviton2 / $CC
+  arm_container:
+    # We use the arm64 images at http://ghcr.io/ruby/ruby-ci-image .
+    image: ghcr.io/ruby/ruby-ci-image:$CC
+    # Define the used cpu core in each matrix task. We can use total 16 cpu
+    # cores in entire matrix. [cpu] = [total cpu: 16] / [number of tasks]
+    cpu: 8
+    # We can request maximum 4 GB per cpu.
+    # [memory per task] = [memory per cpu: 4 GB] * [cpu]
+    memory: 32G
+  env:
+    CIRRUS_CLONE_DEPTH: 50
+    optflags: '-O1'
+    debugflags: '-ggdb3'
+    RUBY_PREFIX: /tmp/ruby-prefix
+    RUBY_DEBUG: ci rgengc
+    RUBY_TESTOPTS: >-
+      -q
+      --color=always
+      --tty=no
+    matrix:
+      CC: clang-12
+      CC: gcc-11
+  id_script: id
+  set_env_script:
+    # Use `make_cmd` instead of `make` environment variable,
+    # because the `make` environment variable is used in rubygems,
+    # and some rubygems tests fail in this case.
+    # https://github.com/rubygems/rubygems/issues/4921
+    - echo "make_cmd=make -s -j$((1 + $CIRRUS_CPU))" >> $CIRRUS_ENV
```

Comment:
Can't avoid the bug by setting the flag only to `GNUMAKEFLAGS`?

Reply:
Good idea! Perhaps `MAKEFLAGS` might be better than `GNUMAKEFLAGS` I will try it.

My memo: [the official doc](https://www.gnu.org/software/make/manual/html_node/Options_002fRecursion.html). I was told it in the past. https://github.com/rake-compiler/rake-compiler/pull/192#issuecomment-817365325 .

Reply:
While `V=1` is a trick written in Makefile and common.mk and probably portable, `-j` is the GNU make specific option.

Reply:
`-s` might be GNU make specific too.

Reply:
OK. Now I understand the `GNUMAKEFLAGS` might be better to emphasize the flags are GNU make specific.